modules/end-user-guide/partials/proc_configuring-an-existing-workspace-to-use-intellij-idea.adoc

```diff
@@ -0,0 +1,15 @@
+[id="configuring-an-existing-workspace-to-use-intellij-idea_{context}"]
+= Configuring an existing workspace to use IntelliJ IDEA
+
+You can configure an already existing workspace to use IntelliJ IDEA as the editor in the {prod-short} dashboard.
+
+.Procedure
+
+. Navigate to the {prod-short} dashboard -> *Workspace* -> *Workspace Configuration* -> the *Editors*
```

Comment:
The new dashboard does not have Editors tab yet.

The way to configure intelij is Devfile -> add into components
```
- type: cheEditor
  id: che-incubator/intellij-community/2020.2.2
```

Reply:
Thanks, updated